dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties

```diff
@@ -16,7 +16,7 @@
 
 description=A secure, REST administrative connector that can be used locally or remotely using any Java \
 developer kit. This connector enables remote access from a Java client or directly through an HTTPS call. \
-It is protected by a single administrator role and requires SSL to keep the communication confidential. \
+The administrator and reader roles protect the connector and it requires TLS to keep the communication confidential. \
```

Comment:
Small suggestion,
Change,
The administrator and reader roles protect the connector and it requires TLS to keep the communication confidential.
to,
The administrator and reader roles protect the connector. The connector requires TLS to keep the communication confidential.

Reply:
@ManasiGandhi - Will make the change.